.gitignore

```diff
@@ -77,6 +77,7 @@ tags
 ### Composer ###
 composer.phar
 /vendor/
+composer.lock
```

Comment:
I have no strong feelings about it but https://getcomposer.org/doc/02-libraries.md#lock-file